packages/backend/src/workers/provider/LlamaCppPython.spec.ts

```diff
@@ -66,6 +68,10 @@ const DummyImageInfo: ImageInfo = {
   engineId: 'dummy-engine-id',
 } as unknown as ImageInfo;
 
+const podmanConnection: PodmanConnection = {
+  getVMType: vi.fn().mockResolvedValue(VMType.WSL),
```

Comment:
I think if you set a mockResolvedValue here, it will be reset when you call `vi.resetAllMocks()` in the beforeEach